Emby.Server.Implementations/Library/MediaStreamSelector.cs

```diff
@@ -95,26 +84,27 @@ public static class MediaStreamSelector
             return stream?.Index;
         }
 
-        private static IEnumerable<MediaStream> GetSortedStreams(IEnumerable<MediaStream> streams, MediaStreamType type, string[] languagePreferences)
+        private static IEnumerable<MediaStream> GetSortedStreams(IEnumerable<MediaStream> streams, MediaStreamType type, IReadOnlyList<string> languagePreferences)
         {
             // Give some preference to external text subs for better performance
-            return streams.Where(i => i.Type == type)
+            return streams
+            .Where(i => i.Type == type)
```

Comment:
```suggestion
            return streams
            .Where(i => i.Type == type)
```

Indentation seems off?